### ISSUE TITLE
Use single connection when configured with passtrough

### ISSUE THREAD

**Issue by angrycoding** [Motion-Project/motion#1146]
My config uses passtrough, so two streams used instead of single one. How motion initiates the connection at the moment (I dont know why):
1. Motion connects to the low res stream and doing a handshake (options, describe, setup, play).
2. Motion sends out teardown.
3. Then it doesn't even wait for receiving OK from camera stream, what it does instead is: opening up two another connections to the low ang high stream.

Here is the problem: I use reverse proxy, so my camera is connecting to my proxy and waits for the motion connection. So I dont have 3 connections at the time.

What i need is either:

1. preffered. Eliminate unnecessary first connect (which is i believe is used to obtain information about the stream or so on). Just use one connection for lo stream and another for hi stream.

2. If first cannot be done, I need motion to wait until OK will be received and then either close socket and connect again, or use same connection to start initialization sequence from scratch (options, describe, setup, play).

To summarize:

- use only one single connection at the time and never try to open new connection to the same stream without closing already opened connection.

- eliminate redundant first connect for probing

- dont close socket if there will be future work with it, use teardown instead

If you can point me to the code where this happens, I can try to fix it myself and submit merge request

**Comment by tosiara** [comment]
Motion has never been tested using any kind of reverse proxy. This is not supported
RTSP code is located here: https://github.com/Motion-Project/motion/blob/master/src/netcam_rtsp.c

**Comment by angrycoding** [comment]
Will close it 'till investigation
